### PR TITLE
Move some things around

### PR DESCRIPTION
**Pull request by tmonck**
Moving some of the startup stuff into the projects that they are adding.

### CODE REVIEW

File: src/AIAD.Api/Startup.cs [hunk 1]
```diff
@@ -45,8 +39,8 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             #region JWT
-            var jwtAppSettings = this.Configuration.GetSection("Jwt").Get<JwtAppSettings>();
-            services.AddSingleton<JwtAppSettings>(jwtAppSettings);
+            services.Configure<JwtAppSettings>(Configuration.GetSection("Jwt"));
+            var jwtAppSettings = services.BuildServiceProvider().GetRequiredService<JwtAppSettings>();
```
Comment (superkarn) [inline src/AIAD.Api/Startup.cs]: This throws a runtime exception because `JwtAppSettings` isn't registered.

File: src/AIAD.Api/Startup.cs [hunk 2]
```diff
@@ -45,8 +39,8 @@ public void ConfigureServices(IServiceCollection services)
...
             services.AddScoped<IJwtService, JwtService>();
```
Comment (superkarn) [inline src/AIAD.Api/Startup.cs]: Not moving this to `AddServicesProjectDependencies()`?

**Review by superkarn — CHANGES_REQUESTED** [review]

File: src/AIAD.Api/Startup.cs (hunk 1 above)
Comment (tmonck) [reply src/AIAD.Api/Startup.cs]: Resolved

File: src/AIAD.Api/Startup.cs (hunk 2 above)
Comment (tmonck) [reply src/AIAD.Api/Startup.cs]: I missed this service. I've moved it now.

**Pull request merged**